trade-finance-manager-ui/server/api.js

```diff
@@ -79,12 +85,16 @@ const getDeals = async (queryParams, token) => {
     if (response.data) {
       return {
         deals: response.data.deals,
-        count: response.data.pagination.totalItems,
+        pagination: response.data.pagination,
       };
     }
     return {
       deals: [],
-      count: 0,
+      pagination: {
+        totalItems: 0,
+        currentPage: queryParams.page,
+        totalPages: 0,
```

Comment:
Could the default `totalItems` and `totalPages` be constants? e.g `DEFAULT_...`

Reply:
Personally I don't think there's much value in this - this is just want we return when there is no response data, so we would never want to change these to be non-zero. I think leaving it inline makes the code more descriptive.